PFOffer.sol

```diff
@@ -183,11 +184,12 @@ contract PFOffer {
     }
 
     function getOneTimePayment() noEther {
-        if (msg.sender != contractor || now < dateOfSignature + 3 weeks )
+        if (msg.sender != contractor || now < dateOfSignature + 3 weeks || oneTimeCostsPayed )
             throw;
 
         if (!contractor.send(oneTimeCosts))
             throw;
+        oneTimeCostsPayed = true;
```

Comment:
Even though it's just a `send` and not a `call`, so probably gas stipend would not be enough to do anything bad maybe move `oneTimeCostsPaid = true;` before the `if (!contractor.send(oneTimeCosts))` ?

Just like we did here: https://github.com/slockit/DAO/pull/200